index_matroska.md

```diff
@@ -39,22 +39,22 @@ fullname="Dave Rice"
 .# Abstract
 
 This document defines the Matroska audiovisual data container structure, including definitions of its structural elements,
-as well as its terminology, vocabulary, and application.
+terminology, vocabulary, and application.
 
 This document updates RFC 8794 to permit the use of a previously reserved Extensible Binary Meta Language (EBML) Element ID.
 
 {mainmatter}
 
 # Introduction
 
-Matroska is an audiovisual data container format. It was derived from a project called [@?MCF],
+Matroska is an audiovisual data container format. It was derived from a project called [@?MCF]
 but diverges from it significantly because it is based on EBML (Extensible Binary Meta Language) [@!RFC8794],
 a binary derivative of XML. EBML provides significant advantages in terms of future format extensibility,
 without breaking file support in parsers reading the previous versions.
 
-First, it is essential to clarify exactly "What an Audio/Video container is", to avoid any misunderstandings:
+To avoid any misunderstandings, it is essential to clarify exactly what an audio/video container is:
```

Comment:
I know this has been in there forever, but maybe use "…exactly what an audiovisual data container is" in order to be consistent with the wording in the previous paragraph.

Reply:
This was one of the RFC editor questions about which one we should use for consistency. I think audiovisual is more generic (includes subtitles). "audio/video" should be kept only when those apply.

I did not make such editorial changes yet, as they may have their own opinion on the subject.

Reply:
Fixed in #827